src/Microsoft.ML.Ensemble/Trainer/Binary/EnsembleTrainer.cs

```diff
@@ -51,6 +52,9 @@ public sealed class Arguments : ArgumentsBase
             [Argument(ArgumentType.Multiple, HelpText = "Base predictor type", ShortName = "bp,basePredictorTypes", SortOrder = 1, Visibility = ArgumentAttribute.VisibilityType.CmdLineOnly, SignatureType = typeof(SignatureBinaryClassifierTrainer))]
             public IComponentFactory<TScalarTrainer>[] BasePredictors;
 
+            [Argument(ArgumentType.Multiple, Visibility = ArgumentAttribute.VisibilityType.EntryPointsOnly)]
```

Comment:
Adding this separetely for now so that it doesn't interfere with tests and command line usage of the existing `BasePredictor` argument